### PR TITLE
Product-detail-radio-buttons-refactor_CU-nx9ufq

### PR DESCRIPTION
**Pull request by a-wilkes**

### CODE REVIEW

**Comment by a-wilkes** [comment]
Task linked: [CU-nx9ufq Product detail radio buttons refactor](https://app.clickup.com/t/nx9ufq)

**Comment by lpboucher** [comment]
Task linked: [CU-nx9ufq Product detail radio buttons refactor](https://app.clickup.com/t/nx9ufq)

File: src/app/modules/products/components/product-detail/product-detail.component.html [hunk 1]
```diff
@@ -13,18 +13,16 @@ <h1 class="product-detail-content-title">{{ product.name }}</h1>
                 </div>
                 <form [formGroup]="optionsForm" class="product-detail-selection-form">
```
Comment (lpboucher) [inline src/app/modules/products/components/product-detail/product-detail.component.html:14]: there is a form tag here and one below, we can only use one and nest everything within it to keep things semantically clean.

(minor adjustment)

Comment (lpboucher) [reply src/app/modules/products/components/product-detail/product-detail.component.html:14]: done

**Review by lpboucher — APPROVED** [review]
Nice, looks good

**Pull request merged**